### PR TITLE
chore(classic-ui/images): add a code snippet

### PR DESCRIPTION
**Pull request by gforcada**
That showcases how to generate a responsive image with it `srcset`.

----
📚 Documentation preview 📚: https://plone6--1678.org.readthedocs.build/

### CODE REVIEW

File: docs/classic-ui/images.md
Comment (stevepiercy) [inline docs/classic-ui/images.md]: I ran `meta` and blackified your code. 😜 

```suggestion
tag = scale_util.picture("image", scale="larger", picture_variant="large")
```

File: docs/classic-ui/images.md
Comment (stevepiercy) [inline docs/classic-ui/images.md]:
```suggestion
To generate a `picture` tag, use the following code.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Thanks for this change. I made a couple of suggestions.

Also I realized that we were not using the correct inline literal syntax for `picture`, which I fixed, and I also added a link to MDN. I never knew this HTML5 tag existed until now, and I just blipped over it.

**Comment by stevepiercy** [comment]
I'd like at least one member of the @plone/classicui-team to review as well.

**Review by stevepiercy — DISMISSED** [review]
LGTM, but it needs another pair of eyes from the @plone/classicui-team. @MrTango @petschki @yurj

File: docs/classic-ui/images.md
Comment (petschki) [inline docs/classic-ui/images.md:302]: Maybe we can add the template implementation here too?

snippet:

```
<div tal:define="scale_view context/@@images">
  <img tal:replace="structure python:scale_view.picture('image', scale='larger', picture_variant='large'" />
</div>
```

**Review by petschki — CHANGES_REQUESTED** [review]
This looks good! see my comment

File: docs/classic-ui/images.md
Comment (gforcada) [reply docs/classic-ui/images.md:302]: Added!

**Review by petschki — DISMISSED** [review]

File: docs/classic-ui/images.md
Comment (yurj) [reply docs/classic-ui/images.md:302]: Can be useful to remember that this can be done on brains too? Just pointing to `plone/app/contenttypes/browser/templates/listing_summary.pt` as an example. 

In general, the receipt for rendering images html code is "get the @@images view on the context you need (object or brain) and use `picture' or `tag` method of the view, using the object as the first parameter if the view has not been already took using it as context". This is my understanding on how it works.

Do tag support all the img tag attributes? In some particular cases, a query to the catalog can retrieve all the info about the original image and let people create a custom html snippet.

File: docs/classic-ui/images.md
Comment (ale-rt) [inline docs/classic-ui/images.md:308]: What about this version that does that without defining the scale view in a parent element?
```suggestion
<tal:picture define="
             scale_view context/@@images;
           "
           replace="structure python:scale_view.picture('image', scale='larger', picture_variant='large'"
/>
```

Comment (yurj) [reply docs/classic-ui/images.md:308]: `tal:picture` if the result is a `<picture>` tag? just cosmetic, nothing important.

File: docs/classic-ui/images.md
Comment (stevepiercy) [inline docs/classic-ui/images.md]:
```suggestion
The same can be done from a template.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Use . at end of sentences.

**Review by stevepiercy — APPROVED** [review]

File: docs/classic-ui/images.md
Comment (stevepiercy) [reply docs/classic-ui/images.md:308]: This comment appears to be resolved, so I will click that button now, so we can merge.

**Comment by stevepiercy** [comment]
Will merge when green.

**Pull request merged**
